FitNesseRoot/DbFit/AcceptanceTests/JavaTests/InformixTests/FlowMode/InformixMode/SetUp/content.txt

```diff
@@ -0,0 +1,3 @@
+!|dbfit.InformixTest|
+
+|Connect|localhost:16466|dbfit|DBFIT|dbfit_ifx|
```

Comment:
We seem to have a mix of approaches going on. e.g. Postgres uses only`Connect` in both stand-alone and flow mode suites. Can we extract this out to another issue/PR?